docs/changelog.rst

```diff
@@ -0,0 +1,6 @@
+*********
+Changelog
+*********
+
+.. include:: rendered_changelog.txt
```

Comment:
For this to work you need to add the generation to the `conf.py` file as well right?

Reply:
I'm not sure what you mean by that. I was thinking of a release flow where the maintainer would do `towncrier` before release so that the changelog is updated.

Can you expand on this?

Reply:
ok, that works, but if this file doesn't exist the doc build will fail.